cluster-autoscaler/clusterstate/utils/types.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
I am confused now. I understood from your earlier comment that I should move it here?

> This is not a part of any api. Lets move the whole file to ./clusterstate/utils/